### DISCUSSION TITLE
Fn lock on specific keys?

### DISCUSSION THREAD

**Discussion by ggjiza** [seerge/g-helper#1021] [Ideas]
The idea is that some Fn-lock functionality would be nice while also is using some Fn keys as usual. Some situations where this could be helpful:
quick access to keys backlight while using F12 normally for some applications.
Having F9 and F10 set as external monitor brightness control (hot keys for another software) while F8&7 (fn-locked) for laptop brightness.

Yeah it's gets daunting forgetting to toggle fn lock turning and off the trackpad instead of getting less blinded by the monitor...

**Comment by seerge** [comment]
I think it will make whole thing complicated, did you know that you know that you can also controll brightness with Ctrl+M1/M2 and backlitght with Shift+M1/M2 ?
